### ISSUE TITLE
Community branch plugin is not showing up the SonarQube results on Bitbucket while doing the pull request

### ISSUE THREAD

**Issue by preddymm** [mc1arke/sonarqube-community-branch-plugin#639]
**Describe the bug**
Community branch plugin is not showing up the scanned sonar results on Bitbucket page for the pull request decoration.
I have identified the following errors in SonarQube logs.
2022.07.05 20:36:00 ERROR ce[AYHQE6iXJLUVHMEVerG5][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Execution of task class com.github.mc1arke.sonarqube.plugin.ce.pullrequest.PullRequestPostAnalysisTask failed
com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.client.BitbucketException: The field 'link' must be a valid HTTP or HTTPS URL
        at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.client.BitbucketServerClient.validate(BitbucketServerClient.java:210)
        at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.client.BitbucketServerClient.uploadReport(BitbucketServerClient.java:140)
        at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.BitbucketPullRequestDecorator.decorateQualityGateStatus(BitbucketPullRequestDecorator.java:102)
        at com.github.mc1arke.sonarqube.plugin.ce.pullrequest.PullRequestPostAnalysisTask.finished(PullRequestPostAnalysisTask.java:160)

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/38227191/177418992-7669cc3e-a197-4988-b3a2-c2720099f9b2.png)
![image](https://user-images.githubusercontent.com/38227191/177422540-8929cbfc-f9c6-422a-adfa-953e7883481f.png)

**Software Versions**
 - SonarQube Version: [8.9.7]
 - Plugin Version: [sonarqube-community-branch-plugin-1.8.1]

**Additional context**
Basically what I am trying to say is Bitbucket Insight API is not working and it is throwing the errors though pull request decoration is failed or passes.
I am using the Bitbucket Data Center with the version 7.21.1
Please fix it ASAP which will really a good feature that is missing on your plugin

**Comment by preddymm** [comment]
> You've not provided enough details to allow this to be replicated:
> 
> * Have you set the Sonarqube server URL in your Sonarqube configuration to be something other than 'localhost'?

**Comment by preddymm** [comment]
Yes, I did replaced with SonarQube URL. What are the additional details that you are looking for to debug this issue?

**Comment by preddymm** [comment]
@mc1arke Any update?

**Comment by Orteko** [comment]
Note that we're having similar issues with PR analysis on bitbucket cloud using sonarqube 9.5 via the docker image "mc1arke/sonarqube-with-community-branch-plugin:9.5-community".

In our case, we have the URL configured and aren't seeing any errors in logs that I can find.

Analysis does run, and fails the build plus we are seeing excerpts from sonar in a PR:

![image](https://user-images.githubusercontent.com/990906/185348543-d9302204-fc2b-40b5-8036-944639b4cfee.png)

However what we're not seeing is the summary banner at the top of a PR - a banner however does display when on the default 'source' tab in bitbucket and not when viewing a PR:

![image](https://user-images.githubusercontent.com/990906/185348776-b76c7e51-1f66-4fc9-8e7b-ce7609deb47b.png)

**Comment by mc1arke** [comment]
@Orteko Are you sure that banner is coming from the Sonarqube instance with this plugin installed? It doesn't look like one the plugin produces.

**Comment by Orteko** [comment]
> @Orteko Are you sure that banner is coming from the Sonarqube instance with this plugin installed? It doesn't look like one the plugin produces.

Hmm interesting - we did have a trial of sonarqube cloud going at one point, maybe there is still some config from that floating around.

I'll do some digging.

**Comment by mc1arke** [comment]
Closing since there have been no recent updates.
